decoder/semantic_tokens.go

```diff
@@ -37,21 +41,45 @@ func (d *PathDecoder) SemanticTokensInFile(filename string) ([]lang.SemanticToke
 	return tokens, nil
 }
 
-func (d *PathDecoder) tokensForBody(body *hclsyntax.Body, bodySchema *schema.BodySchema, parentModifiers []lang.SemanticTokenModifier) []lang.SemanticToken {
+func (d *PathDecoder) tokensForBody(ctx context.Context, body *hclsyntax.Body, bodySchema *schema.BodySchema, parentModifiers []lang.SemanticTokenModifier) []lang.SemanticToken {
 	tokens := make([]lang.SemanticToken, 0)
 
 	if bodySchema == nil {
 		return tokens
 	}
 
+	if bodySchema.Extensions != nil {
+		ctx = icontext.WithExtensions(ctx, bodySchema.Extensions)
+		if bodySchema.Extensions.Count {
+			if _, ok := body.Attributes["count"]; ok {
+				// append to context we need count provided
+				ctx = icontext.WithActiveCount(ctx)
+				log.Printf("Found Expression: ")
+			}
+		}
+	}
+
 	for name, attr := range body.Attributes {
+
+		log.Printf("Found: %q / %v+", name, bodySchema.Extensions)
 		attrSchema, ok := bodySchema.Attributes[name]
 		if !ok {
-			if bodySchema.AnyAttribute == nil {
-				// unknown attribute
-				continue
+			if bodySchema.Extensions != nil && name == "count" && bodySchema.Extensions.Count {
+				attrSchema = &schema.AttributeSchema{
+					Description: lang.MarkupContent{
+						Kind:  lang.MarkdownKind,
+						Value: "**count** _optional, number_\n\nThe distinct index number (starting with 0) corresponding to the instance",
+					},
```

Comment:
```suggestion
```
Given that this is purely for semantic tokens, I think we don't need the description here?